sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Fixed a race condition that could result in a memory/thread leak for `BulkExecutor` instances (and their corresponding `cosmos-daemon-BulkExecutor-*` thread). - See [PR 31082](https://github.com/Azure/azure-sdk-for-java/pull/31082)
 
 #### Other Changes
+* Enable replica validation by default - See [PR 31159](https://github.com/Azure/azure-sdk-for-java/pull/31159) 
```

Comment:
May be worth adding some more context on this feature explicitly in the release notes.

Reply:
@kirankumarkolli  any suggestions here?

Reply:
Removed from the changelog